README.md

```diff
@@ -256,6 +256,12 @@ Atom Packages
 - atom-beautify (automatically fix code mistakes for you)
 - multi-wrap-guide (add multiple wrap guides at different column widths)
 
+(If using React.js)
+- js-hyperclick 
```

Comment:
I am guessing these are atom plugins for react, great idea.  Can you add a brief description of what each one of these is in parentheses to make it flow with the plugins above?